### PR TITLE
add tenant label to offerings and plans

### PR DESCRIPTION
**Pull request by evyaffe**
Tenant label is required for offerings and plans in addition to the current setup in which broker labels only has it.

### CODE REVIEW

File: pkg/types/base.go [hunk 1]
```diff
@@ -61,6 +61,14 @@ func (b *Base) GetLabels() Labels {
 	return b.Labels
 }
 
+func (b *Base) ExtractTenantLabelValue(tenantLabelKey string) []string {
```
Comment (eyalnathan) [inline pkg/types/base.go:64]: not used anymore, right?

File: storage/interceptors/broker_create_catalog_interceptor.go [hunk 2]
```diff
@@ -141,3 +148,13 @@ func brokerCatalogAroundTx(ctx context.Context, broker *types.ServiceBroker, fet
 
 	return nil
 }
+
+func addTenantToLabels(labels types.Labels, tenantKey string, tenantValue []string) types.Labels {
```
Comment (eyalnathan) [inline storage/interceptors/broker_create_catalog_interceptor.go:152]: this is not adding, it is setting. wouldn't setTenantLabel be a better name?

Comment (eyalnathan) [reply storage/interceptors/broker_create_catalog_interceptor.go:152]: or even setLabelValue, you pass in the key as well

File: pkg/types/base.go (hunk 1 above)
Comment (evyaffe) [reply pkg/types/base.go]: removed

File: storage/interceptors/broker_create_catalog_interceptor.go (hunk 2 above)
Comment (evyaffe) [reply storage/interceptors/broker_create_catalog_interceptor.go]: changed

**Comment by eyalnathan** [comment]
@evyaffe can we close this one? we can keep the branch around if we think we might need it at some point

**Pull request closed without merging**